### ISSUE TITLE
OpenID Connect Error

### ISSUE THREAD

**Issue by the-markus** [kromitgmbh/titra#214]
Hi!

When trying to add OpenID Connect Authentication to titra, we first get the following error:

```
{"line":"414","file":"oauth_server.js","message":"Error in OAuth Server: Unexpected OAuth service oidc","time":{"$date":1724073804937},"level":"warn"}
```

After restarting the application, the following error is thrown when trying to login using the configured OIDC:

```
{"line":"414","file":"oauth_server.js","message":"Error in OAuth Server: Cannot read properties of undefined (reading 'error')","time":{"$date":1724073932969},"level":"warn"}
```

**Comment by faburem** [comment]
thanks for your PR to fix this issue, it has been merged and is currently released as version 0.99.10 🥳

**Comment by the-markus** [comment]
Thanks for the merge :)
